src/app/components/BibPage/BibPage.jsx

```diff
@@ -206,34 +221,25 @@ class BibPage extends React.Component {
     const bNumber = record && record.idBnum ? record.idBnum : '';
     const marcRecordLink = bNumber ? 'https://catalog.nypl.org/search~S1?' +
       `/.b${bNumber}/.b${bNumber}/1%2C1%2C1%2CB/marc` : '';
+    const searchURL = createAPIQuery({});
```

Comment:
We probably don't need this function. We can call `basicQuery(this.props)` directly instead.